### DISCUSSION TITLE
Data Retenstion and Data stored location of metrics

### DISCUSSION THREAD

**Discussion by dyadav-dinesh** [NetApp/harvest#2974] [General]
We are planning to deploy the harvest in our company, for that we need few things.
1.  What is the default retention period for Prometheus metrics?
2. Where it save metrics data location ?
3. can we change the default retention period ?

**Comment by rahulguptajss** [comment]
@dyadav-dinesh Are you inquiring about the Harvest-provided container installation as listed [here](https://netapp.github.io/harvest/24.05/install/containers/)? If yes, then as of the 24.05 release, the default retention period for this workflow is 15 days, as set in Prometheus. The data is stored in a Docker volume named `harvest_prometheus_data`. If you wish to change the retention period, please follow the guidelines [here](https://netapp.github.io/harvest/24.05/install/containers/#customize-prometheuss-retention-time).

Prometheus is open source, so you can customize it as provided by Prometheus. If you are setting up Prometheus yourself, you may find more details [here](https://prometheus.io/docs/prometheus/latest/installation/).
